django_pgschemas/utils.py

```diff
@@ -381,15 +185,14 @@ def clone_schema(base_schema_name, new_schema_name, dry_run=False):
 
     # check if the clone_schema function already exists in the db
     try:
-        cursor.execute("SELECT 'clone_schema'::regproc")
+        cursor.execute("SELECT 'public.clone_schema(text, text, boolean, boolean)'::regprocedure")
     except ProgrammingError:  # pragma: no cover
         _create_clone_schema_function()
         transaction.commit()
 
     try:
         with transaction.atomic():
-            sql = "SELECT clone_schema(%(base_schema)s, %(new_schema)s, TRUE)"
-            cursor.execute(sql, {"base_schema": base_schema_name, "new_schema": new_schema_name})
+            cursor.callproc("clone_schema", [base_schema_name, new_schema_name, True, False])
```

Comment:
What would happen if `denishpatel/pg-clone-schema` function changes its signature? 

Maybe to avoid breaking stuff would be better to download a specific version of their branch?

Reply:
The process of downloading a new version of the script and shipping in a new version of the package would be manual, and subject to all tests passing. Should they change their signature or exception handling, the tests won't pass and I or whoever is desiring to cut a release will know and review.

So to clarify, the script is not downloaded automatically for users using pgschemas. It's a manual process performed by the maintainers of pgschemas in preparation of a new release.

Reply:
this is great!